modules/nf-core/epang/split/meta.yml

```diff
@@ -0,0 +1,50 @@
+name: "epang_split"
+description: write your description here
+keywords:
+  - phylogeny
+  - phylogenetic placement
+  - sequences
+tools:
+  - "epang":
+      description: "Massively parallel phylogenetic placement of genetic sequences"
+      homepage: "https://github.com/Pbdas/epa-ng"
+      documentation: "https://github.com/Pbdas/epa-ng/wiki/Full-Stack-Example"
+      tool_dev_url: "https://github.com/Pbdas/epa-ng"
+      doi: "https://doi.org/10.1093/sysbio/syy054"
+      licence: "['GNU Affero General Public License v3.0']"
+
+input:
+  # Only when we have meta
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - refaln:
+      type: file
+      description: reference alignment in any supported format including phylip and fasta, may be gzipped
+      pattern: "*"
```

Comment:
You should add a selection of patterns here, like https://github.com/nf-core/modules/blob/b265b4ff6a35b133b963b4eaddfca0ffb3395236/modules/nf-core/custom/tabulartogseagct/meta.yml#L20

Reply:
I'm not so fond of that for fasta files since they can have such a large number of different suffixes, depending on user preferences.

Reply:
Moreover, many file formats are supported.

Reply:
I think there's precedent for very long lists of extensions:

```
https://github.com/nf-core/modules/blob/014209cabf9120e1a89014e95ad687170ca9c37a/modules/nf-core/rgi/main/meta.yml#L25
```

... but little precedent for none, so I still think extensions are a sensible thing here. Happy to be corrected if one of the other maintainers disagrees.

Reply:
There is precedent, admittedly by myself, but accepted by, IIRC, @jfy133 not so long ago: https://github.com/nf-core/modules/blob/master/modules/nf-core/hmmer/eslreformat/meta.yml

Reply:
NGL I've not been very consistent with that 😂, nowadays I generally prefer having extensions though, even if it's a long list